src/jit/compiler.h

```diff
@@ -183,6 +183,13 @@ class LclSsaVarDsc
 
 typedef JitExpandArray<LclSsaVarDsc> PerSsaArray;
 
+enum RefCountState
+{
+    RCS_INVALID, // not valid to get/set ref counts
+    RCS_EARLY,   // early counts for struct promotion and passing
+    RCS_NORMAL,  // normal ref counts (from lvaMarkRefs onward)
```

Comment:
This comment looks like it is truncated:
"and passing"...?

Reply:
um, struct passing. I will clarify.